xml/gl.xml

```diff
@@ -41042,6 +41047,11 @@ typedef unsigned int GLhandleARB;
                 <command name="glFramebufferShadingRateEXT"/>
             </require>
         </extension>
+        <extension name="GL_EXT_fragment_shading_rate" supported="gl|glcore">
+            <require>
```

Comment:
That change is not quite what I meant - you cannot have multiple \<extension> blocks for the same extension. If you make this edit:

```suggestion
            <require api="gl">
```

and then remove lines 41049-41050 (\</extension> ... \<extension>), then there will be two \<require> blocks for the same \<extension>, one of which is tagged just for GL (unfortunately github does not appear to allow me to make suggestions which would change multiple lines at once, unlike gitlab).